rtv/subreddit_page.py

```diff
@@ -60,6 +62,12 @@ def refresh_content(self, order=None, name=None):
         if not self.term.loader.exception:
             self.nav = Navigator(self.content.get)
 
+    def reload_footer(self):
```

Comment:
I think this would be a great place to use a property, e.g.

```python
@property
def footer(self):
    if self.reddit.user is None:
        return docs.FOOTER_SUBREDDIT
    else:
        return docs.FOOTER_SUBREDDIT_LOGGED_IN
```

Then all you would need to do is change the other pages to lowercase, e.g. for the subscription page

```python
footer = docs.FOOTER_SUBSCRIPTION
```

and update the base class to use ``self.footer`` instead of ``self.FOOTER``. I agree that uppercase no longer makes sense, and it was probably a questionable decision when it was added in the first place. If you want to change ``self.HEADER`` to lowercase at the same time I'm on board with it.